src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -756,7 +783,7 @@ public void given_an_event_when_directionDueDate_is_empty_then_task_should_start
         String caseIdForTask = UUID.randomUUID().toString();
         EventInformation eventInformation = EventInformation.builder()
             .eventInstanceId(UUID.randomUUID().toString())
-            .eventTimeStamp(LocalDateTime.now().minusDays(1))
+            .eventTimeStamp(eventTimeStamp)
```

Comment:
Any specific reason for changing this.

Reply:
Just removing duplicates when extracted _LocalDateTime.now().minusDays(1)_